components/src/md3/Grades/Grades.tsx

```diff
@@ -1,42 +1,47 @@
 import React from 'react';
-import { ViewProps } from 'react-native';
+import { Platform, ViewProps } from 'react-native';
 import { Avatar, useTheme } from 'react-native-paper';
 
+// Define the prop types for the Grade component
 export type GradeProps = ViewProps & {
     label: string;
-    color?: string;
+    fontColor?: string;
     backgroundColor?: string;
     size?: number;
 };
 
-export type FixedGradeProps = Omit<GradeProps, 'label' | 'color' | 'backgroundColor'>;
+// Omit certain props from GradeProps to create FixedGradeProps
+export type FixedGradeProps = Omit<GradeProps, 'label' | 'fontColor' | 'backgroundColor'>;
 
+// GradeBase component definition
```

Comment:
When I said in-code "JSDocs", I was looking for specifically the [JSDocs](https://jsdoc.app/) that can show documentation in tooltips when our adopters are using our components in their IDEs. They follow a certain format. You can refer to our existing components, say [List Item Tag](https://github.com/etn-ccis/blui-react-native-component-library/blob/dev/components/src/core/list-item-tag/list-item-tag.tsx), to see how this is done.